kitsune/products/jinja2/products/documents.html

```diff
@@ -32,7 +32,7 @@
     <div class="firefox-download-button hidden">
       <div class="documents-download-buttons download-buttons">
         <div class="download-firefox">
-          <a href="https://www.mozilla.org/firefox/download/thanks/" class="download-button"
+          <a href="https://www.mozilla.org/firefox/download/thanks/?utm_source=support.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=non-fx-button&amp;utm_content=body-download-button" class="download-button"
```

Comment:
I didn't think ampersands in URLs in a tags needed to be escaped to `&amp;`. I could be wrong on that thought.

Reply:
It's probably no longer required - I think modern browsers will compensate. We do it on bedrock, so my muscles are trained. 🤷‍♂️

Reply:
Great! Works for me then. 👍